gatsby-node.js

```diff
@@ -5,13 +5,3 @@
  */
 
 // You can delete this file if you're not using it
-const path = require('path')
-
-exports.onCreateWebpackConfig = ({ actions, loaders, plugins, stage }) => {
-  actions.setWebpackConfig({
-    resolve: {
-      modules: ['src', 'node_modules'],
```

Comment:
We still need this `modules` here, no wonder the build failed in Netlify. 😅

Reply:
I'm sorry, my bad @zainfathoni  👍